websites/A/Apple/metadata.json

```diff
@@ -29,8 +33,8 @@
 		"www.icloud.com"
 	],
 	"version": "2.3.1",
-	"logo": "https://i.imgur.com/rBAusA5.png",
-	"thumbnail": "https://i.imgur.com/D5b7fQ5.jpg",
+	"logo": "https://i.imgur.com/w84AMlL.png",
+	"thumbnail": "https://i.imgur.com/83wQlSy.jpeg",
```

Comment:
```suggestion
	"thumbnail": "https://i.imgur.com/MZecshr.jpg",
```
![image](https://user-images.githubusercontent.com/53608074/198343240-ecffd01b-b2a8-475f-b4a0-374e7e0f1ad3.png)

The thumbnail doesn't fit. We should use this one instead:
![image](https://user-images.githubusercontent.com/53608074/198344520-dcda4fed-42ed-4359-84f2-e9942dad2f95.png)